### PR TITLE
HttT units: Review needed.

### PR DESCRIPTION
**Pull request by doofus-01**
These are some updates to some units in HttT.  For now it is Ashevier, Lisar, and the Sea Orc.  They aren't the greatest animations, but I think they good enough for units that aren't encountered often.  Sea Orc is a joke unit, sorry if I ruined the joke.  If I didn't ruin the joke, I can come up with a portrait (based off some mainline unit, this guy doesn't really deserve a new or commissioned portrait).  Let me know, either way.

### CODE REVIEW

**Comment by doofus-01** [comment]
Argh, I've screwed things up, the first two commits (changelog and aquatic) are not supposed to be in here, they are in another PR.  I'll try to figure this out, but ... damn ...

**Comment by Vultraz** [comment]
No need. We can deal with that. In fact, we'll likely just merge it together.

**Comment by doofus-01** [comment]
Thanks, Vultraz.  If some revision is needed, please make note on the forums.

File: data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg [hunk 1]
```diff
@@ -60,9 +64,15 @@
...
         [/filter_attack]
         start_time=-200
+        swoosh_start_time=-50
         [frame]
-            image="units/human-queen-staff-[1~4].png:[100*3,75]"
+            image="units/human-queen.png:100,units/human-queen-attack.png:200,units/human-queen.png:1"
         [/frame]
+        [swoosh_frame]
+            image="units/human-queen-swoosh.png:100,misc/blank-hex.png:1"
+	    auto_vflip=no
```
Comment (Vultraz) [inline data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg]: Only problem I see if the bad indent here. Other than that it looks fine.

Comment (doofus-01) [inline data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg]: Great!  What do you (or whoever is dealing with HttT) think about the need for Sea Orc portrait, does that guy still have dialogue?

Comment (Vultraz) [inline data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg]: Yeah, he has some lines. A portrait for him might be nice, but it's not a high priority.

**Comment by CelticMinstrel** [comment]
> In fact, we'll likely just merge it together.

Uhh. They're unrelated. Don't group them in the same merge.

Anyway, all we need to do to merge this is `git cherry-pick f97f3ab` (once your PR is fetched to a local computer), so it's not a big deal that there's extra stuff in it. We can easily just grab the actual relevant bit.

**Comment by doofus-01** [comment]
If the swooshes need to go to images/halo, that's fine, but if it is only for that one unit, maybe it should stay where it is?

I can work on a portrait, but it will be a limited endeavor, since this is a niche image.

**Comment by Vultraz** [comment]
I think he meant in HttT/images/halo, not data/core/images/halo

**Comment by CelticMinstrel** [comment]
You might have a point since it's a campaign-specific image. I'd wait to see what zookeeper has to say though.

File: data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg (hunk 1 above)
Comment (GregoryLundberg) [inline data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg]: Currently he uses the unit image. It's small and, frankly, a bit funky looking.

As a Sea Orc, he has a line when he changes (turn 4), and another when he's killed.

And, to be honest, I never noticed he actually changed appearance until I was working on the campaign. So I tried to get the camera on him when he changes from a normal Orc to a Sea Orc so there's at least a small chance the change on-screen will draw the player's eye.

Comment (ln-zookeeper) [inline data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg]: Arguably, he shouldn't switch from the default Orcish Archer portrait to the unit image. Yet if the portraits stays the same through the transformation, it's kind of unclear what the whole schtick is about. Some kind of frankenstein portrait combining his first portrait (which can be something other than Orcish Archer, if it makes frankensteining easier) with some merfolk equipment seems like the best route, if one can find an even remotely natural-looking combination. I can look into it and see if I can find something that could work.

**Comment by ln-zookeeper** [comment]
The swooshes can certainly go to HttT/images/units, but is there really a need for them to be separate images in the first place? I didn't check the timings, but I presume the idea is to display the swoosh for only part of the duration of the corresponding unit frame, but it seems like it'd be more straightforward and simpler to just clone another unit frame and bake the swoosh into it.

**Comment by doofus-01** [comment]
The swooshes don't really need to be separate, I guess, but as separate frames, they can have a directional offset.  Does it cause some issue?

**Comment by ln-zookeeper** [comment]
The swooshes aren't an issue as such, it just seems like the kind of thing which won't really end up being used for anything so they seem like an unnecessary complication. They could have directional offsets, true, but it seems like it could be a couple of pixels at best before they'd disconnect from the weapon. If you want to keep them then that's fine.

**Comment by Vultraz** [comment]
Manually merged f97f3ab into master in 83bc21c5d8db.

**Pull request closed without merging**